### ISSUE TITLE
New way of describing performances [M42 + M43]

### ISSUE THREAD

**Issue by pasqLisena** [DOREMUS-ANR/doremus-ontology#17]
No more `efrebroo:F31_Performance`(or maybe just as a node, with F31 p9 consists of various M42).

The standard way of describing performance now is the use of `mus:M42_Performed_Expression_Creation`. All performances create a `mus:M43_Performed_Expression`

> Why create M42 and M43? because not all performances perform a preexisting Work (think of improvisation, but also of some traditional music) and when they do they may perform it in a such a way that you cannot say they "just" performed the Work (a typical case, though not unique, is jazz performance: My Favorite Things by John Coltrane is not "just" a performance of the original from The Sound of Music. 
-- @pierrechoffe 

Discussion topics:

1. `efrbroo:F31_Performance` can create an expression? Therefore, why do we want to create a specific **M42** class? Is it to distinguish performance that creates expressions or not? Besides the fact that this is really argue-able, this is not necessary at all, the whole graph enables already to make this distinction. So what is the purpose of M42?

2. Similarly, why do we want to create a **M43**, if this is an expression? What is the fundamental distinction of a M43 in terms of being an expression? Are there different type of expressions (fundamentally)? If this is just to indicate that it results from a performance, well, again the graph says it already, no need to sub-type.

Discussion started in #15

**Comment by pierrechoffe** [comment]
@pasqLisena 
>efrbroo:F31_Performance can create an expression? Therefore, why do we want to create a specific M42 class

No, F31_Performance can *not* create an Expression. Please check the ["CRM, FRBRoo, PRESSoo" schema](https://drive.google.com/drive/folders/0B55dxt4olm-DRVhST0JLZUJrWVk) which makes things very clear (although M42 is not in the schema yet...).
There is *only one* event in FRBRoo that can create Expressions, it is F28_Expression_Creation (and of course its subclasses). Therefore, if we want to create a class that is both a performance and an expression-creation, it has to be a subclass of both F31 and F28.

>Similarly, why do we want to create a M43, if this is an expression? What is the fundamental distinction of a M43 in terms of being an expression? Are there different type of expressions (fundamentally)? If this is just to indicate that it results from a performance, well, again the graph says it already, no need to sub-type.

If we use this reasoning for FRBRoo, we can as well suppress F24_Publication_Expression and F26_Recording :smile:
To be consistent with the FRBRoo modeling, we created M24_Track and M43_Performed_Expression (both subclasses of F22). Besides the consistency matter, there was another reason (from my point of view) : for M24, we clearly create an entity that is familiar to people using ontologies for music. In the case of M43_Performed_Expression, this is a very important distinction in the world of libraries, between "notated music" (F22) and "performed music" (M43). If we use F22, which technically would be quite possible, we lose this important distinction. So, in both cases, we are trying to make things clear to *the people who will use our ontology* and use concepts they know. True, our ontology could be more concise, but this would be at the expense of its usability.

**Comment by rtroncy** [comment]
> No, F31_Performance can not create an Expression. Please check the [CRM, FRBRoo, PRESSoo schema](https://drive.google.com/file/d/0B4s5dBwAmhJsN0tQUTVSbi1Wc3c/view) which makes things very clear (although M42 is not in the schema yet...).
There is only one event in FRBRoo that can create Expressions, it is F28_Expression_Creation (and of course its subclasses). Therefore, if we want to create a class that is both a performance and an expression-creation, it has to be a subclass of both F31 and F28.

I have fixed the link to point to the correct pdf file (not the modeling folder). This schema does not say that a F31 cannot create an expression. Where is this formal restriction formulated? Is it the fact of not being a E65 (or broadly a E63) that prevents this? If this is the case, then joining F31 and F28 will also break this rule and enable some activities to create an expression will be equally wrong.

> If we use this reasoning for FRBRoo, we can as well suppress F24_Publication_Expression and F26_Recording

F24 and F26 **are** expressions. There are already the results of some (creation) events. We are not talking about the same thing. If you're after name consistency, then your ```mus:M42_Performed_Expression_Creation``` should actually be ```mus:M42_Performance_Event```

I agree for the existence of M43 but this is not an argument of conciseness. The valid argument is that this sub-type can have additional properties.

**Comment by pasqLisena** [comment]
> 1. why do we want to create a specific M42 class?

I think here the point is **to have a class that extends both F31 and F28**.
The only alternative solution is adding properties to F31, but I do not see particular advantages in this solution (and I think it is also less elegant).

> 2. why do we want to create a M43?

I see in the distinction between the two classes also the possibility to distinguish them (i.e. in queries). But probably this could be done seeing the properties (so if it is linked to a Creation Event or to a Performance one).


@pierrechoffe you are planning to draw a schema about the new way of express performances?

**Comment by pierrechoffe** [comment]
>@pierrechoffe you are planning to draw a schema about the new way of express performances?

You can find a detailed one [here](https://drive.google.com/file/d/0B4s5dBwAmhJsSXhFX2ZDWHV2cFk/view?usp=sharing)

**Comment by pierrechoffe** [comment]
@pasqLisena 
>I think here the point is to have a class that extends both F31 and F28.

Yes, this is how things work in CRM & FRBRoo. Btw, this solution was brought up by Patrick Le Boeuf (joker :smile:) as early as December 2015.

@rtroncy 
>This schema does not say that a F31 cannot create an expression. Where is this formal restriction formulated?

There is no formal restriction formulated. But the rules of construction in CRM & FRBRoo are so: F28 is a subclass of Creation and of Production, itself a subclass of Modification. Modification and Creation are subclasses of Activity.
The specificity of F28 is that instead of producing or creating a material object it creates a concept (the Expression).
F31 does not create anything because it is not a subclass of Creation. I know this may sound strange, but this is how it works in CRM & FRBRoo. Concepts inherited from broader concepts.
So M42_Performed_Expression_Creation inherits from F31_Performance, which is an activity, and from F28_Expression_Creation which also is an activity. M42 is also a Modification and a performance can be both a modification of an existing Expression and a creation of sth new: the resulting Expression. *My Favorite Things* by John Coltrane modifies the original song (duration 3mn, for voice and orchestra) in creating a new Expression (duration 20mn, for saxophone and rhythm section)

>F24 and F26 are expressions. There are already the results of some (creation) events. We are not talking about the same thing. If you're after name consistency, then your mus:M42_Performed_Expression_Creation should actually be mus:M42_Performance_Event

I don't understand. F28_Expression_Creation *is* an event. As well as F31 Performance, and all the Modification, Creation, Production, Move, Assignment classes. Actually almost no event classes have "event" in their name. mus:M42_Performed_Expression_Creation takes its name from efrbroo:F28_Expression_Creation. Calling it mus:M42_Performance_Event would be error prone (proximity with F31_Performance).
M43 is the result of the M42 event, it is an Expression, and for the reasons exposed earlier is a specific notion that "talks" to librarians.

**Comment by rtroncy** [comment]
I miss the fact that E65 is *also* an activity (those CRM guys have a very different view than the DOLCE guys that primarily distinguish events from activities). Thanks for the [Concerto VI Paganini-Spohr example](https://drive.google.com/file/d/0B4s5dBwAmhJsSXhFX2ZDWHV2cFk/view), this helps.

Regarding the naming of M42, and since we are sub-classing F28, you have:
 - ```efrbroo:F29_Recording_Event``` that R21_created a ```efrbroo:F26_Recording```
 - ```efrbroo:F30_Publication_Event``` that R24_created a ```efrbroo:F24_Publication_Expression```

Therefore, I suggest to have:
```mus:M42_Performance_Event``` that R17_created a ```mus:M43_Performance_Expression```

@pierrechoffe Can you explain me the differences in terms of semantics between ```efrbroo:R21``` (not properly defined), ```efrbroo:R24``` and ```efrbroo:R17```? I would personally always use ```efrbroo:R17```.

**Comment by pasqLisena** [comment]
+1 for the renaming proposed by @rtroncy .

I'm seeing the schema. `mus:U59_is_performed_expression_of` (or `mus:U59_is_performed_expression_of`, I think you are still evaluating the final name)  is not yet in the turtle file. I see this property is connecting 2 expressions, that is not usual in our ontology. 
Maybe it could be better connect M42 to F22 instead, but they are yet connected by performance plan.
Moreover we connect M43 to F15 with R3.
So basically, we need `mus:U59`?


I am a bit lost in this thread. Definitively, we agree on the existence of M42 an M43?

**Comment by pierrechoffe** [comment]
@rtroncy yes you are right, things are not very clear. As an example E69_Death is an event (but not called so) while F29_Recording_Event is an activity, not an event :smile: This si all very confusing.

The [Concerto Vl Paganini-Spohr example](https://drive.google.com/file/d/0B4s5dBwAmhJsSXhFX2ZDWHV2cFk/view) was based on a document pointed to by Marie at the Rome AG, see page 12 of the [Variations3 document](http://www.dlib.indiana.edu/projects/variations3/docs/v3FRBRreport.pdf). I think it is interesting to compare both schemas.

I have been thinking thouroughly about the naming of M42 and M43, I would suggest keeping things as they are for the following reasons:

* The examples you mention already show there is no consistency : F26_Recording should have been F26_Recording_Expression
* I created M29_Editing (an activity, subclass of F28) and asked Patrick if we should change its name to Editing_Event, his answer was "no" (btw, I also asked whether we should create an Editing_Work, his answer was "no", therefore I deleted the proposed E39_Editing_Work)
* As already mentioned, M42_Performance_Event is confusing with F31_Performance. Furthermore it doesn't say the important thing, which is that it creates an expression.
* Since we create these new classes, it should be very clear to the users (library folks) that they correspond to the concept of "performed music" they are familiar with. So, the words "Performed Expression" should appear in both M42 and M43.

>Can you explain me the differences in terms of semantics between efrbroo:R21 (not properly defined), efrbroo:R24 and efrbroo:R17? I would personally always use efrbroo:R17.

I would do the same :smile: (use R17 everywhere), but for whatever reason they wanted to have a narrower definition for F20 and F21

**Comment by pierrechoffe** [comment]
@pasqLisena the property is in the turtle file, with a different number (U54) since a did a refactoring of property numbers.
U54_is_performed_expression_of aims to link the "notated music" and the "performed music", which may have differences (remember the example of Bach's concerto BWV 1060). It would correspond to what you see in the Philharmonie records, where they list the recordings associated with a musical piece, but this would be more precise (they list albums, while this property links to the performed piece).
I think we should have a conversation about this approach, I am presently trying to find all possible ways we can answer accurately any request from the users. Maybe we do not need these properties, maybe we need other properties I haven't thought of, you tell me.
Another point is I would like to create easy parallels with schema.org (where you would just need alignments between one prop and another).

Coming back to U54, it links M43 with F22 (two Expressions) and is a subproperty of P130_shows_features_of, so it expresses a similarity.

**Comment by pierrechoffe** [comment]
@pasqLisena @rtroncy 

>Definitively, we agree on the existence of M42 an M43?
I think we do, and I hope we agree on my naming.
The concept of performed expression is established in the library community, it solves problems in the description of recordings AND it is the solution to our problems of modeling *traditional music* :smile:

**Comment by rtroncy** [comment]
For clarification:
* Resolution 1: we will create the ```mus:M42``` and ```mus:M43``` classes. There seem to be a consensus on this.
* Resolution 2: we will systematically use the relationship ```efrbroo:R17``` instead of sub-properties that have the same semantics but seem to only restrict the domain and/or range

@pierrechoffe, you wrote:
> while F29_Recording_Event is an activity, not an event

Activities *are* events. Let's be precise. We do not have consensus **yet** in the naming. I'm not after purity and I don't care about opinions of people (even if this comes from Mr LeBoeuf) if this comes without argument. I don't feel that ```F26_Recording``` is ambiguous and don't convey the fact that this is an expression despite not being in the name. I also agree that ```M29_Editing``` is not ambiguous and could stay as is without being called ```M29_Editing_Event```.

> As already mentioned, M42_Performance_Event is confusing with F31_Performance. Furthermore it doesn't say the important thing, which is that it creates an expression.

I don't think ```M42_Performance_Event``` is really confusing, in particular, since we will never instantiate the F31 class. The second part of your argument is worst: a name is just a name and not the definition! The concept has a definition which carries out its meaning. We don't have argument on the naming of M43. We seem to agree to name it: ```mus:M43_Performance_Expression```.

> Another point is I would like to create easy parallels with schema.org (where you would just need alignments between one prop and another).

This should definitively **NOT** be your goal. You're starting point is the data you have, not an hypothetical target model which is changing.

The remaining question is about the ```mus:U59``` (or ```mus:U54```, I'm confused) property. We used to have properties between Work. You're now entering into what seems to me a dangerous slope of duplicating what you said between Work for Expression. This definitively needs more thoughts and examples, and probably a new issue.

**Comment by rtroncy** [comment]
An alternative for naming M42 could also to use the gerund, which would yield: ```M42_Performing```

**Comment by pierrechoffe** [comment]
@rtroncy I don't really understand why you consider M42_Performed_Expression_Creation a problem. This name says exactly what this class is : an Expression creation (as its inheritance from F28_Expression_Creation implies) which happens to be a Performed_Expression (as implied by the inheritance from F31_Performance). This is the creation of a performed expression. Création d'une Expression Interprétée.
Since we seem to all agree on the Performed Expression name, why does this (x + Creation) create a problem?
I agree that the name is just a name and the real definition is the scope note. I hear this said repeatedly at each CRM SIG Meeting. But why? Because people take the property name as granted when they seem clear. To me Performance Event (or Performing Event) is *not* clear and it is error prone. The only clear name is Performed Expression Creation. Is it too long? too librarian-oriented? :smile:

**Comment by rtroncy** [comment]
I can exactly take the reverse arguments that yours. How many classes in FRBRoo or even in DOREMUS have ```Expression_Creation``` in their name apart from F28? None ... and there are many sub-classes of F28. Still, nobody complains that the sub-classes of F28 don't have ExpressionCreation in their name despite the fact that they create Expression. What's wrong with the gerund?

**Comment by pierrechoffe** [comment]
:smile: the wrong thing with gerund or with performance is that (1) they don't say what they do (what is the difference between "performance" and "performance_event"?), (2) they look very similar to F31 when they are actually very different (especially for someone coming from FRBRoo, who is used to a performance being nothing alse than a performance), (3) performing event may sound strange to English-speaking people, but I'm not sure, we would have to ask them. To me, a "performing event" is relatively rare and more like "activité de spectacle" than like "interprétation/exécution", [check here](http://www.linguee.fr/anglais-francais/traduction/performing+events.html).
Now your turn : what's wrong with performed expression creation (création d'une expression performée) ? :smile:

**Comment by rtroncy** [comment]
The gerund perfectly conveys the idea that there is an activity (per its definition) and this is perfect English. I'm not suggesting to add the term *Event*. Editing, Recording, Performing are all very clear to barely anyone you will ask in the street. The similarity with F31 has little importance given that F31 will not be used.

I don't immediately understand what is a *performed expression creation*. As with any compound names, you always wonder in which order you have to put the terms and what is ultimately created. It is harder to memorize. No other sub-classes of F28 have expression creation in their names, etc. I have produced many arguments

**Comment by pierrechoffe** [comment]
May I ask our friends at CRM SIG (Steven Stead, he is the one who cares for the right naming) ?

**Comment by rtroncy** [comment]
The more opinions, the better. It is strange though that you don't think to ask first and foremost to the DOREMUS partners.

**Comment by pierrechoffe** [comment]
No it is not strange, first because he is an English speaker and second because he is much much much more into these reasonings than us all. He is a wise man, his opinion is generally accepted by all at CRM and as I said, when it comes to naming things, he is the one that talks.

**Comment by delahousse** [comment]
Même en anglais restez cool
Ça à l air d avancer c est le principal
Bonne journée
Jean

Jean Delahousse

delahousse.jean@gmail.com
+33601224855

**Comment by pierrechoffe** [comment]
@delahousse on est cool Jean, chacun présente ses arguments :smile: C'est intéressant parce qu'on voit les choses depuis des perspectives différentes.

**Comment by pasqLisena** [comment]
The only thing in which there is still not agreement is the name for property M42.
Candidates:
  1. Performed Expression Creation 
  2. Performance Event
  3. Performing

**Comment by rtroncy** [comment]
I would vote for 3. If there is time in tomorrow's modeling meeting, it would be good to poll the participants. / cc @ccecconi

**Comment by pasqLisena** [comment]
In order to better mark the difference with F31, the modelling group decided to keep M42 as is (Performance Expression Creation).
